subprojects/docs/src/docs/release/notes.md

```diff
@@ -125,9 +125,12 @@ in the next major Gradle version (Gradle 5.0). See the User guide section on the
 
 The following are the newly deprecated items in this Gradle release. If you have concerns about a deprecation, please raise it via the [Gradle Forums](https://discuss.gradle.org).
 
-<!--
-### Example deprecation
--->
+### Deprecated usage of path separator characters in names
+
+We deprecated the usage of the following characters in domain object names such as project or task names:
```

Comment:
This should explicitly call out the case in `settings.gradle` where users call `include 'foo/bar'` and tell them to use `include 'foo:bar'` instead. Otherwise this warning might be non-actionable for them. Maybe our documentation for multi-projects isn't good enough, but this comes up *a lot*.

Reply:
major bummer this didn't get documented. this fundamentally broke my build and i had to do a bunch of googling and i still can't figure out why renaming my projects doesn't work anymore

```include 'path:projecta' 'path:projectb'
project.(':path:projecta').name = 'projecta'
project.(':path:projectb').name = 'projectb'```

Reply:
This is unrelated to this change (or should be). The code you showed could be made clearer, but it still should work afaik.

Nothing should break, we only deprecated at this point. 

Please open a bug report with a reproducible example.

Reply:
Regarding the original comment by @oehme about documenting this better. Use this issue for further discussion: #3008